src/types.ts

```diff
@@ -70,6 +70,7 @@ export interface OptionContract {
 }
 
 export interface OptionOrderParams extends OptionContract {
+    payPremium?: boolean | null, // Set to `True` if the user will pay the premium using stablecoin. Set to `False` to pay the premium using their collateral
```

Comment:
I think you can remove the `| null` since quantity also has the `?` but doesn't have `| null` next to it